.github/workflows/push_tests.yml

```diff
@@ -12,53 +16,111 @@ env:
   MKL_NUM_THREADS: 8
   PYTEST_TIMEOUT: 600
   RUN_SLOW: yes
+  PIPELINE_USAGE_CUTOFF: 50000
 
 jobs:
-  run_slow_tests:
+  setup_torch_cuda_pipeline_matrix:
+    runs-on: docker-cpu
```

Comment:
The push tests should definitely still run on GPU, the only difference here should maybe just be that the RUN_SLOW flag is set to False instead of True. So we run the fast tests but on GPU this time